tooling/noir_js/src/debug.ts

```diff
@@ -110,7 +110,8 @@ function getCallStackFromLocationNode(
 
     callStack = location_tree[callStack.parent];
   }
-  return result;
+  // Reverse since we explored the child nodes first
+  return result.reverse();
```

Comment:
I switched to just reverse at the end of the loop after pushing the stack rather than unshifting for every element. `unshift` has a complexity of O(n) where `n` is size of the array vs an O(1) push per element and just reversing a single time at the end of the loop. cc @sirasistant